storage/client/pom.xml

```diff
@@ -98,7 +112,35 @@
   </dependencies>
 
   <build>
+
+    <sourceDirectory>src/main/java</sourceDirectory>
```

Comment:
Do we exclude the sample codes for running the record mode?

Reply:
Sample are in the test directory, so they should get picked up with other classes.

Reply:
the sample goes under src/samples/java. I did not get how I got pick up from test directory.